mapApp/static/mapApp/js/vis/visHelpers.js

```diff
@@ -9,32 +12,57 @@ var labels = {
 }
 // Define scales
 // map p_type to appropriate colors
-var colorScale = d3.scale.ordinal()
+var colorScale = d3.scaleOrdinal()
     .domain(["collision", "nearmiss", "hazard", "theft"])
     .range([iconColors.collision, iconColors.nearmiss, iconColors.hazard, iconColors.theft]);
-var weekdayScale = d3.scale.quantize()
-    .domain([0,1,2,3,4,5,6])
+
+var weekdayScale = d3.scaleQuantize()
+    .domain([0,6])
     .range(moment.weekdaysShort());
 
-// Set date formats to local language
-moment.locale(LANGUAGE_CODE)
+// var timeScale = d3.scaleTime()
```

Comment:
Can we remove these two lines?

Reply:
this just got moved to the top of the file, thought it made more sense for imports/set up stuff!